doc/admin-guide/plugins/multiplexer.en.rst

```diff
@@ -59,6 +60,7 @@ Example remap.config::
     map http://www.example.com/a http://www.example.com/ @plugin=multiplexer.so @pparam=host1.example.com
     map http://www.example.com/b http://www.example.com/ @plugin=multiplexer.so @pparam=host2.example.com
     map http://www.example.com/c http://www.example.com/ @plugin=multiplexer.so @pparam=host1.example.com @pparam=host2.example.com
+    map http://www.example.com/c http://www.example.com/ @plugin=multiplexer.so @pparam=host1.example.com @pparam=host2.example.com @pparam=proxy.config.multiplexer.skip_post_put=1
```

Comment:
I know it's just an example, but having the same 2 inbounds in remap.config is invalid. Can you update `http://www.example.com/c` to http://www.example.com/d` ?

Reply:
@randall my bad, fixed